### ISSUE TITLE
Question on labels and on dates of uploaded emails

### ISSUE THREAD

**Issue by caspertone2003** [jay0lee/got-your-back#298]
Hi,
I am unable to understand why #297 was closed. #296 is not addressing my question.
I read the documentation and searched through the issues, and was unable to find answer to my question. So, let me try again. My two questions:

- Is there any way to just bakcup labels and restore labels, no email down/up?

- Are restored emails receiving their original receive timestamp or do they receive the date of restore?

Please bear me as I was unable to find answers to them.

Thanks!

**Comment by meksone** [comment]
It's not possibile to backup just the labels; for the second question, restored email retain the original date.
